lib/sign_in/idme/configuration.rb

```diff
@@ -71,7 +71,19 @@ def jwt_decode_algorithm
       end
 
       def jwt_decode_public_key
-        OpenSSL::PKey::RSA.new(File.read(Settings.idme.oauth_public_key))
+        IdmePublicKey.public_keys ||= fetch_idme_public_keys
+      end
+
+      def fetch_idme_public_keys
+        uri = URI(Settings.idme.oauth_public_key_url)
+        response = Net::HTTP.get_response(uri)
```

Comment:
I think we want some handling when this is not a `200` response, or when we don't find any keys, should at least log but maybe return `nil` as well. Some form of response so that it won't be memoized in the `jwt_decode_public_key` function above (so that we'll keep calling it and attempting to find the key if we call this again)